package.json

```diff
@@ -52,6 +49,17 @@
   },
   "peerDependencies": {
     "react": "^16.9.0",
-    "react-dom": "^16.9.0"
+    "react-dom": "^16.9.0",
+    "@instructure/canvas-theme": "^8.2.1",
+    "@instructure/emotion": "^8.2.1",
+    "@instructure/ui-heading": "^8.2.1",
+    "@instructure/ui-icons": "^8.2.1",
+    "@instructure/ui-modal": "^8.2.1",
+    "@instructure/ui-svg-images": "^8.2.1",
+    "@instructure/ui-text": "^8.2.1",
+    "@instructure/ui-themeable": "^7.5.1",
+    "@instructure/ui-themes": "^8.2.1",
+    "@instructure/ui-spinner": "^8.2.1",
+    "@instructure/ui-view": "^8.2.1"
```

Comment:
So the exact versions are not exactly specified, the  ^ sign means it will be compatible with anything within the minor version change.
 I think for strict and exact version, it would just be "x.y.z", without any sign,
while adding the ~ sign is for flexibility with patch changes  and adding the * sign is for flexibility with major changes.

Reply:
Sorry my example should be `>= 7`, I think that works in packages.json doesn't it?

Reply:
Yes it should, I would update in the next push

Reply:
Magic, thanks.